### DISCUSSION TITLE
VroidSDKforUEで認証を通さない場合の挙動

### DISCUSSION THREAD

**Discussion by papicochan** [pixiv/vroid-sdk-developers#126] [Q&A] [answered]
Unreal Engine5.3にてVroidSDKを使用しています。
認証を通さず、もしくは認証後VRMファイルのロードに失敗した状態でレベルに遷移するとTポーズのVroidの初期の女の子キャラクターが表示されますが、これを正しくアニメーションするモデルに差し替えるにはどうすればよろしいでしょうか

**Comment by ytomi-pixiv** [comment, marked as answer]
いただいた状態※ですと、たとえばモデルのD&Dによる差替えでアニメーションの再生ができるかと思います。
下記ドキュメントをご参照ください。
ただし、ロードできないモデルの場合はクラッシュするか、アニメーションリターゲットに失敗してTPoseで出てきます。現状ロードできないモデルに関sh知恵は、対応可能な範囲を増やすべく取り組んでおりますので、いましばらくお待ちください。
https://vroid.notion.site/VRoid-SDK-for-Unreal-Engine-21321b5e31b247e99600f270b3414d4e#:~:text=%E3%81%8C%E5%87%BA%E6%9D%A5%E3%81%BE%E3%81%99%E3%80%82-,Drag%20And%20Drop%E3%81%A7%E3%81%AE%E3%83%A2%E3%83%87%E3%83%AB%E5%B7%AE%E3%81%97%E6%9B%BF%E3%81%88%E3%81%AB%E5%AF%BE%E5%BF%9C%E3%81%99%E3%82%8B%E3%81%AB%E3%81%AF,-Drag%20And%20Drop

※未認証の場合、または認証後のモデルロードに失敗した場合

**Reply by papicochan** [reply]
ご教授ありがとうございました。
デフォルトキャラのまま(D&Dをせず)動く状態で出てくるようにすることは現状手段がないということですね。
